examples/msg/README.doc

```diff
@@ -45,7 +45,7 @@ In addition to the fully documented example of @ref
 MSG_ex_asynchronous_communications, there are several other examples
 shipped in the archive:
   
- - <b>Basic asynchronous communications</b>. 
+ - <b>Basic asynchronous and synchronous communications</b>. 
```

Comment:
Make a new section in the doc called "Basic synchronization", don't merge it with asyncs